samples/star/build.gradle.kts

```diff
@@ -95,7 +95,7 @@ dependencies {
   implementation(libs.okio)
   implementation(libs.retrofit)
   implementation(libs.retrofit.converters.moshi)
-  implementation(libs.leakcanary.android)
+  debugImplementation(libs.leakcanary.android)
```

Comment:
@ZacSweers weird I don't remember adding this

Reply:
I did, fixed it along the way